srcts/src/shiny/bind.ts

```diff
@@ -120,13 +131,25 @@ const bindingsRegistry = (() => {
       })
       .join("\n");
 
+    const message = `The following ${
+      duplicateIds.size === 1 ? "ID was" : "IDs were"
+    } repeated across input and output bindings:\n${duplicateIdMsg}`;
+
+    if (status === "warning") {
+      return {
+        status: "warning",
+        event: new ShinyClientMessageEvent({
+          headline: "Shared input/output IDs found",
+          message: message,
```

Comment:
To match the one below
```suggestion
          message,
```